README.md

```diff
@@ -3202,6 +3203,7 @@ _Software written in Go._
 - [go-selfupdate](https://github.com/sanbornm/go-selfupdate) - Enable your Go applications to self update.
 - [gobrew](https://github.com/cryptojuice/gobrew) - gobrew lets you easily switch between multiple versions of go.
 - [gobrew](https://github.com/kevincobain2000/gobrew) - Go version manager. Super simple tool to install and manage Go versions. Install go without root. Gobrew doesn't require shell rehash.
+- [gobuilder](https://github.com/gvcgo/gobuilder) - An enhancement for official go build.
```

Comment:
Has no documentation on how to use all the features that are mentioned in the readme.
Replacing `$` with `#` is unncessary when "$(git ...)" is a bash command which gets replaced even before it gets passed to the executable as an argument.
As per awesome-go guidelines 80% code coverage is required, your readme has no links to code coverage.
It is unclear what this tool does without running it, and it is only a 3 week project at v0.0.5, awesome-go requires a stable v1.0.0 release.